packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -640,6 +640,10 @@ section.modal-panel {
     background-image: url('/images/icon-service-private-network.svg');
   }
 
+  &[data-name^='Firefox Better Web with Scroll'] {
```

Comment:
> I have a scroll better web account, but I'm not sure how to get it to show up on my devices list

I cheated a bit and updated the `data-name` property on the row to `Firefox Better Web with Scroll` and it works.

<img width="630" alt="Screen Shot 2020-04-27 at 1 41 17 PM" src="https://user-images.githubusercontent.com/1295288/80403085-fd9dd600-888c-11ea-891d-13ff3a5f727c.png">